docs/Project.toml

```
@@ -7,12 +7,13 @@ CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 CellListMap = "69e1c6dd-3888-40e6-b3c8-31ac5f578864"
 ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DelaunayTriangulation = "927a84f5-c5f4-47a5-9785-b46e178433df"
 DiffEqCallbacks = "459566f4-90b8-5000-8ac3-15dfb0a30def"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
-DynamicSumTypes = "5fcdbb90-de43-509e-b9a6-c4d43f29cf26"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
 DrWatson = "634d3b9d-ee7a-5ddf-bec9-22491ea816e1"
+DynamicSumTypes = "5fcdbb90-de43-509e-b9a6-c4d43f29cf26"
```

```suggestion
LightSumTypes = "f56206fc-af4c-5561-a72a-43fe2ca5a923"
```

This package is actually deprecated in favour of `LightSumTypes.jl`